samples/step-by-step/sample.md

```diff
@@ -7,100 +7,190 @@ redirects:
 - nservicebus/nservicebus-step-by-step-guide
 ---
 
-In this sample shows a very simple ordering system that
+This sample illustrates most of the basic NServiceBus concepts in a step-by-step fashion. You can either download the code from the link above and run it, or you can build it yourself following the instructions here.
```

Comment:
i still dont see the value of personal voice here. eg the above could be 

> This sample illustrates most of the basic NServiceBus concepts in a step-by-step fashion. Either download the code from the link above and run it, or build it with the instructions.

It is even more concise